lib/travis/api/v3/access_control/generic.rb

```diff
@@ -96,6 +96,14 @@ def settings_visible?(settings)
       repository_visible?(settings.repository)
```

Comment:
There are 3 1-line methods here that do the same thing. Do we think there might be different logic here in the future? I think the generic access control might have too much knowledge -> settings and env_vars have a repository. I haven't dug through and seen the places calling these methods, but perhaps they should just call repository_visible? w/ the repo directly? I'm not sure yet, just a potential suggestion for discussion.